app/tests/unit/helpers.tsx

```diff
@@ -1,5 +1,5 @@
 import { DEFAULT_STATE_VALUES } from '@store/constants'
-import { pomodoroStore } from '@store/Pomodoro'
+import { pomodoroStore } from '@store/store'
```

Comment:
Shouldn't all our current files in the `store` folder be placed inside `store/Pomodoro`, as all of them are related?

And then we can have a new `/store/store.ts` which exports all the stores together, which now it would only be `PomodoroStore`.